### PR TITLE
Major Text Change Process

### LINKED ISSUE #21: Major Descriptive Text Changes Process
Currently, the PMIx Standardization Process does not have a formal process for major text changes. For example, significant rewrites of introductory material or movement of chapters.

I would like to propose the following outline for a process (formal language will be in the PR once we agree on the outline):
 * Major Descriptive Text Changes defined as significant changes to descriptive text in the document such as introductory text in a chapter, definitions of terms, re-organization of chapter contents or chapter placement, non-errata level changes to descriptions of APIs including advice to implementors/users. Smaller changes to the text may be considered Errata as proposed in Issue #20 .
 * Text Changes must be filed in as a GitHub PR in the draft state linked to all impacted GitHub Issues until ready for broad review.
 * Once the PR is ready for review, it will move out of the draft state and marked with the "RFC" and "Major Text Change" labels. A Straw Poll Comment is added to the PR and email sent to the mailing list.
 * All concerns must be addressed on the PR. The author may withdraw the PR at any time.
 * Once the discussion has consolidated then the Co-Chairs will add the PR to the agenda for the next quarterly meeting. The label "Eligible" will be added at this time.
 * Author(s) presents the PR for discussion at the ASC quarterly meeting in a formal reading.
 * A formal ASC vote is taken to designate as "Pushed Back" or "First Vote Passed"
   - If "Pushed Back" for minor clarifications, the authors can present an abbreviated reading of just the changes since the last presentation duirng the next quarterly meeting before another first vote is taken. The ASC may request a full reading if the changes since the last vote are deemed too expansive.
 * If passed, the author(s) present it again at the following quarterly meeting (formal reading is not required unless requested).
 * A formal ASC vote is taken to designate as "Pushed Back" or "Accepted"
 * "Accepted" will be merged by the Release Managers.
 * The text changes will be included in the next major release.

Since major text changes are difficult to slowly introduce into the text next to the old language, following a "Provisional acceptance" process won't work. Instead, I propose that we follow a slower process more like the "Elevation to Stable status" since the text change will become stable formal language in the document.

This process follows the informal process we adopted in eary 2020 as documented in the [pdf](https://github.com/pmix/pmix-standard/wiki/files/2020-01-ASC/2020-01-ASC-Meeting.pdf) slides (slide 10) from the [2020 - Q1](https://github.com/pmix/pmix-standard/wiki/ASC-Q1-2020-Meeting#agenda-timeline) Meeting - Jan. 23, 2020.

### PR DESCRIPTION
**Pull request by jjhursey**
* Closes #21

### CODE REVIEW

**Comment by jjhursey** [comment]
Please use emoji reactions ON THIS COMMENT to indicate your position on this proposal.

* You do not need to vote on every proposal
* If you have no opinion, don't vote - that is also useful data
* If you've already commented on this issue, please still vote so
      we know your current thoughts
* Not all proposals solve exactly the same problem, so we may end
      up accepting proposals that appear to have some overlap
This is not a binding majority-rule vote, but it will be a very
significant input into the corresponding ASC decision.

Here are the meanings for the emojis:

* Hooray or Rocket: I support this so strongly that I
      want to be an advocate for it
* Heart: I think this is an ideal solution
* Thumbs up: I'd be happy with this solution
* Confused: I'd rather we not do this, but I can tolerate it
* Thumbs down: I'd be actively unhappy, and may even consider
      other technologies instead
If you want to explain in more detail, feel free to add another
comment, but please also vote on this comment.

File: pmix_governance.tex [hunk 1]
```diff
@@ -1092,5 +1092,124 @@ \subsubsection{Deprecation}\label{deprecation}}
...
+\label{major-text-changes}}
+
+Major Text Changes are defined as significant changes to descriptive text in
+the document such as introductory text in a chapter, definitions of terms,
+re-organization of chapter contents or chapter placement, non-errata level
+changes to descriptions of APIs including advice to implementors/users.
+
+The process for major text changes begins with a GitHub Draft PR while writing the proposal, optionally proceeded by a GitHub Issue for more general discussion and guidance.
+When the GitHub PR is ready for discussion the authors(s) must remove the Draft PR status, add the ``RFC'' and ``Major Text Change'' labels, and add the Straw Poll Comment to it.
+Then the author(s) must send an announcement of the PR to the PMIx community mailing list requesting consideration of the PR as an errata change.
```
Comment (abouteiller) [inline pmix_governance.tex:1108]: as a major text  change

Comment (jjhursey) [reply pmix_governance.tex]: Fixed. Thanks!

**Comment by kathrynmohror** [comment]
Line 610: I suggest changing "All proposed additions to the PMIx Standard enter the PMIx Standard
  under Provisional status. " to something like "All proposed API or attribute additions to the PMIx Standard enter the PMIx Standard under Provisional status. "

Line 719: I suggest changing "Discussion of the PR will continue until all dissenting opinions and
questions have been addressed, upon which the PR may be put forward for
Provisional acceptance. " to something like " Discussion of the PR will continue until all dissenting opinions and
questions have been addressed, upon which the PR may be put forward for formal decision by the ASC."

File: pmix_governance.tex [hunk 2]
```diff
@@ -1092,5 +1092,124 @@ \subsubsection{Deprecation}\label{deprecation}}
...
+\subsubsection{Major Text Changes}%
+\label{major-text-changes}}
+
+Major Text Changes are defined as significant changes to descriptive text in
+the document such as introductory text in a chapter, definitions of terms,
+re-organization of chapter contents or chapter placement, non-errata level
+changes to descriptions of APIs including advice to implementors/users.
+
+The process for major text changes begins with a GitHub Draft PR while writing the proposal, optionally proceeded by a GitHub Issue for more general discussion and guidance.
+When the GitHub PR is ready for discussion the authors(s) must remove the Draft PR status, add the ``RFC'' and ``Major Text Change'' labels, and add the Straw Poll Comment to it.
```
Comment (mike0042) [inline pmix_governance.tex:1107]: author(s)

Comment (jjhursey) [reply pmix_governance.tex]: Fixed. Thanks!

**Comment by jjhursey** [comment]
@mike0042 and @kathrynmohror I addressed your comments in the last two commits. Note that I also rebased this PR onto master to pick up the 'draft' watermark.

The current PDF rendering is here: 
[pmix_governance.pdf](https://github.com/pmix/governance/files/6320469/pmix_governance.pdf)

**Comment by jjhursey** [comment]
PMIx ASC 2Q 2021: May 11 & May 13
 * Passed first vote: 7 yes / 0 no / 1 abstain
 * Eligible for second vote 3Q 2021 ASC Meeting

The new commit is a typo fix found during the reading.

**Comment by jjhursey** [comment]
PMIx ASC 3Q 2021
 * Passed the second vote: 8 yes / 0 no / 1 abstain
 * Will be merged into the next release

**Comment by jjhursey** [comment]
I rebased and merged this PR. It'll be in the v1.5 version of the document.

**Pull request merged**